sdk/communication/azure-communication-common-perf/pom.xml

```diff
@@ -19,6 +19,9 @@
 
   <properties>
     <perf.test.mainClass>com.azure.communication.common.perf.App</perf.test.mainClass>
+
+    <!-- TODO: Remove this once all Javadoc warnings and errors are resolved -->
+    <doclint>all,-missing</doclint>
```

Comment:
Do you have an issue filed so that we can scan for these and keep burning them down (if so, you should add the issue here to make scanning even more).

Reply:
I do now: https://github.com/Azure/azure-sdk-for-java/issues/26454